pkg/eval/str/str_test.go

```diff
@@ -69,6 +73,9 @@ func TestStr(t *testing.T) {
 		That(`str:to-codepoints a`).Puts("0x61"),
 		That(`str:to-codepoints 你好`).Puts("0x4f60", "0x597d"),
 
+		That(`str:to-utf8-bytes a`).Puts("0x61"),
+		That(`str:to-utf8-bytes 你好`).Puts("0xe4", "0xbd", "0xa0", "0xe5", "0xa5", "0xbd"),
```

Comment:
A string can contain bytes that are not part of a valid UTF8 sequence. There should be a test for that case.

Reply:
Ignore my previous comment. While what I wrote is technically true it is also true that the Go runtime already handles this and has its own tests that it is handled correctly. Adding an elvish test for that case wouldn't provide any benefit.